base/applications/taskmgr/taskmgr.h

```diff
@@ -36,15 +36,14 @@ typedef struct
 	BOOL	HideWhenMinimized;
 	BOOL	Show16BitTasks;
 
-	/* Update speed settings */
-	/* How many half-seconds in between updates (i.e. 0 - Paused, 1 - High, 2 - Normal, 4 - Low) */
+	// 0 - Paused, 1 - High, 2 - Normal, 4 - Low
```

Comment:
```suggestion
	/* Update speed settings: 0 - Paused, 1 - High, 2 - Normal, 4 - Low */
```

Reply:
Commenting the values makes sense, we do both agree to that.
But I don't see the reason for commenting a variable which is named as "UpdateSpeed" with a comment "update speed",
and it doesn't make any sense to name an attribute that is part of a struct named "Settings" with a comment named "settings". Both is redundant, and not helpful.

So I used the same comment here, that we do use in releases/0.4.7 to releases/0.4.14:
![image](https://github.com/reactos/reactos/assets/33393466/25e965b9-2917-4cdc-a47f-d4031c1f62df)